### ISSUE TITLE
Load the details into database.

### ISSUE THREAD

**Issue by NavyaNelluri** [NavyaNelluri/Project-TWA#2]
load the job seekers and employer details into respective tables.

**Comment by saichetanveeragandham** [comment]
Its better we have the table structure too at this point

**Comment by NavyaNelluri** [comment]
yeah..for now with the initial google form I just created table structure and updated in ReadMe.md file.

**Comment by harjothkaur** [comment]
Looks good to me

**Comment by Shirisha777** [comment]
which type of tables structure are you thinking
